src/tools/perf/perftest.c

```diff
@@ -332,6 +332,29 @@ static ucs_status_t setup_sock_rte_loopback(struct perftest_context *ctx)
     return UCS_OK;
 }
 
+void backup_local_params(const perftest_params_t *params,
+                         perftest_params_local_backup_t *backup)
+{
+    if (params->super.api == UCX_PERF_API_UCT) {
+        strcpy(backup->dev_name, params->super.uct.dev_name);
+    } else {
+        backup->dev_name[0] = '\0';
+    }
+
+    backup->needed_flags = params->super.flags &
+                           UCX_PERF_TEST_FLAG_ERR_HANDLING;
+}
+
+void restore_local_params(perftest_params_t *params,
+                          const perftest_params_local_backup_t *backup)
+{
+    if (params->super.api == UCX_PERF_API_UCT) {
+        strcpy(params->super.uct.dev_name, backup->dev_name);
+    }
+
+    params->super.flags |= backup->needed_flags;
```

Comment:
shouldn't we assign here instead of append?

Reply:
According to existing logic, they should be or'ed. In fact, this is just sync of error handling enabling from client side to server.